packages/react-reconciler/src/SchedulingProfiler.js

```diff
@@ -59,32 +71,52 @@ if (enableSchedulingProfiler) {
   }
 }
 
-export function formatLanes(laneOrLanes: Lane | Lanes): string {
-  let labels = getLabelsForLanes(laneOrLanes);
-  if (labels != null) {
-    labels = labels.sort().join(',');
-  } else {
-    labels = '';
+const laneLabels: Array<string> = [];
+
+export function getLaneLabels(): Array<string> {
+  if (laneLabels.length === 0) {
+    let lane = 1;
+    for (let index = 0; index < TotalLanes; index++) {
+      laneLabels.push(((getLabelForLane(lane): any): string));
+
+      lane *= 2;
```

Comment:
sorry if dumb question, why are we doubling the lane here each time?

Reply:
I think it's because lanes is a bitmask, so `lane *= 2` essentially means to get the label for the next (lower priority) lane